Debug log total displacement ranges

amimre opened the pull request:
A small addition to log total displacement ranges at debug level.
This helps to keep track of the total ranges in a looped search.

michele-riva reviewed (changes requested):
Some suggestions for improvements.

Could you also add tests?

amimre commented:
I've rebased to the 0.12.0 branch, implemented these changes, and added a simple test.

I wouldn't invest much more time into this, as this is not a priority at all. This is basically just intended as a temporary band-aid to keep track of the displacement ranges, as the current displacements are quite convoluted and not very flexible.

michele-riva commented:
Sounds good. Are you done with the edits? Once you are, can you re-request the review?

michele-riva reviewed (changes requested):
Sorry, still a few things to do

amimre commented on src/viperleed/calc/classes/atom.py:
```diff
@@ -12,12 +12,15 @@
 __created__ = '2019-06-13'
 __license__ = 'GPLv3+'
 
+from dataclasses import field
+from typing import Dict
 import copy
 import logging
```
It already is?

michele-riva replied:
As a string, yes, but we've been sorting on the module name, irrespective of whether it is an `import module` or a `from module import`. I would stick to this, as it makes it easier to group imports. Here it would be `copy`, `dataclasses`, `logging`, `typing`.

michele-riva reviewed (approved):
Looks good, feel free to merge!
